app/views/clone_account_memberships/index.html.haml

```diff
@@ -0,0 +1,18 @@
+= content_for :tabnav do
+  = render partial: "admin/shared/tabnav_users", locals: {secondary_tab: "accounts"}
+= content_for :sidebar do
+  = render :partial => 'admin/shared/sidenav_users', locals: { sidenav_tab: 'users' }
```

Comment:
This is probably copy/paste, but we should try to maintain stylistic consistency within the same files, e.g. using the newer hash syntax and double quotes. `partial: "admin/shared/sidenav_users"`